Projects/Core/Services/KakaoLogin/KakaoLoginClient.swift

```diff
@@ -67,14 +67,11 @@ extension KakaoLoginClient: DependencyKey {
       },
       loginWithKakaoAccount: { @MainActor in
         try await withCheckedThrowingContinuation { continuation in
-          var scopes = [String]()
-          scopes.append("openid")
-          
-          UserApi.shared.loginWithKakaoAccount(scopes: scopes) { oauthToken, error in
+          UserApi.shared.loginWithKakaoAccount { oauthToken, error in
             if let error {
```

Comment:
```suggestion
            if error != nil {
```
error를 if let 바인딩하고 쓰는곳이 없어 아마 워닝으로 알려줄것같은데요.
이렇게 해당 바인딩 값을 사용하지 않으면 적용하는것이 좋을것 같아요.
한번 전체적으로 다른 부분들도 컴파일 워닝을 띄우는곳이 있다면 수정해주시면 좋겠습니다 🙋🏻